.vib/parse/goss/vars.yaml

```diff
@@ -0,0 +1,18 @@
+binaries:
+  - node
+  - parse-server
+  - python
+  - mongosh
+directories:
+  - mode: "0775"
+    paths:
+      - /bitnami/parse
+      - /opt/bitnami/parse
+      - /opt/bitnami/parse/lib
+      - /opt/bitnami/parse/logs
+      - /opt/bitnami/parse/public_html
+      - /opt/bitnami/parse/views
```

Comment:
Unless I'm mistaken, we are not explicitly setting the permissions to the directories added at compilation stage (lib, views, and public_html). I think we could move them to a separate array without the `mode` check.

Reply:
Yes we are:

```
for dir in "$PARSE_BASE_DIR" ...; do
    ...
    configure_permissions_ownership "$dir" -d "775" -f "664" -u "$PARSE_DAEMON_USER" -g "root"
done
```

This block will recursively change the permissions in the folder. With this test, we are checking that those permissions are set properly. Note that those folders are created during the compilation.

Reply:
You are right, we normally do not check subfolders of a recursive permissions change but I agree that, as we already need to check them due to the compilation logic, we can also check the permissions.